api_tests/registrations/views/test_registration_files_list.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from tests.json_api_test_app import JSONAPITestApp
+from api.base.settings.defaults import API_BASE
+from api_tests import utils as api_utils
+from tests.base import ApiTestCase
+from osf_tests.factories import (
+    AuthUserFactory,
+    ProjectFactory,
+    RegistrationFactory
+)
+
+@pytest.mark.django_db
+class TestRegistrationFilesList(object):
+
+    @pytest.fixture(autouse=True)
+    def setUp(self):
```

Comment:
Per discussion, this throws a weird error that we haven't been able to diagnose yet: `Failed: Database access not allowed, use the "django_db" mark, or the "db" or "transactional_db" fixtures to enable it.`